nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -46,6 +46,10 @@ include::modules/eco-resuming-node-maintenance-cr-web-console.adoc[leveloffset=+
 
 include::modules/eco-resuming-node-maintenance-cr-cli.adoc[leveloffset=+2]
 
+[id="about-collecting-data-nmo"]
+== About collecting data for the Node Maintenance Operator
+To collect debugging information about the Node Maintenance Operator, use the `must-gather` tool. For information about the `must-gather` image for the Node Maintenance Operator, see xref:../../support/gathering-cluster-data.adoc#gathering-data-specific-features_gathering-cluster-data[Gathering data about specific features].
```

Comment:
This link doesn't work, but I assume because this link is in another PR, right?

Reply:
@prabinovRedhat the [link](http://file.rdu.redhat.com/avbhatt/TD663/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data) seems to be working fine. Ensure that you are connected to VPN.

Reply:
works